### PR TITLE
Add dynamites output to static analysis to prevent mistakes

### PR DESCRIPTION
**Pull request by provokateurin**

### CODE REVIEW

File: packages/nextcloud/lib/src/nextcloud.openapi.dart [hunk 1]
```diff
@@ -35,8 +38,8 @@ class NextcloudResponse<T, U> {
   String toString() => 'NextcloudResponse(data: $data, headers: $headers)';
 }
 
-class _Response {
-  _Response(
+class RawResponse {
```
Comment (Leptopoda) [inline packages/nextcloud/lib/src/nextcloud.openapi.dart:41]: If wanted we can still hide it from the export.

File: packages/dynamite/lib/src/openapi_builder.dart [hunk 2]
```diff
@@ -1862,3 +1877,16 @@ TypeResult resolveType(
 
   throw Exception('Can not convert OpenAPI type "${schema.toJson()}" to a Dart type');
 }
+
+// ignore: avoid_positional_boolean_parameters
+int sortRequired(final bool a, final bool b) {
```
Comment (Leptopoda) [inline packages/dynamite/lib/src/openapi_builder.dart:1882]: This took me a bit to understand.

Maybe going forward we should add doc comments to those stuff so others can understand easier.

Comment (provokateurin) [reply packages/dynamite/lib/src/openapi_builder.dart:1882]: Yeah, I agree. Dart sort methods are pretty weird to use imo.

File: packages/nextcloud/lib/src/nextcloud.openapi.dart (hunk 1 above)
Comment (provokateurin) [reply packages/nextcloud/lib/src/nextcloud.openapi.dart:41]: Yeah, but I don't think it matters that much.

File: packages/dynamite/lib/src/openapi_builder.dart (hunk 2 above)
Comment (Leptopoda) [reply packages/dynamite/lib/src/openapi_builder.dart:1882]: We don't need to explain sort methods but mentioning it is one is enough.

My head expanded the `sortRequired` into `isSortRequired` and I then expected it have a boolean return type.

Just mentioning it is a sort method for required params would have be it.

Comment (provokateurin) [reply packages/dynamite/lib/src/openapi_builder.dart:1882]: Is `sortRequiredElements` good? It sorts both parameters and properties, so it has to be agnostic.

**Review by Leptopoda — APPROVED** [review]

**Pull request merged**